src/main/java/net/hypixel/nerdbot/command/AdminCommands.java

```diff
@@ -177,6 +178,18 @@ public void getEveryonesNames(GuildSlashEvent event) throws IOException {
         event.getHook().sendFiles(FileUpload.fromData(file)).queue();
     }
 
+    @JDASlashCommand(name = "forcesave", description = "Forcefully save a user's data", defaultLocked = true)
+    public void forceSaveUser(GuildSlashEvent event, @AppOption(description = "The user to save") Member member) {
+        if (NerdBotApp.USER_CACHE.getIfPresent(member.getId()) != null) {
```

Comment:
store this value and then null check it, instead of using the getIfPresent method twice